les/client.go

```diff
@@ -99,7 +106,9 @@ func New(ctx *node.ServiceContext, config *eth.Config) (*LightEthereum, error) {
 		bloomRequests:  make(chan chan *bloombits.Retrieval),
 		bloomIndexer:   eth.NewBloomIndexer(chainDb, params.BloomBitsBlocksClient, params.HelperTrieConfirmations),
 		serverPool:     newServerPool(chainDb, config.UltraLightServers),
+		valueTracker:   lpc.NewValueTracker(lespayDb, &mclock.System{}, requestList, time.Minute, 1/float64(time.Hour), 1/float64(time.Hour*1000)),
```

Comment:
I don't think `1000 * time.Hour` is a reasonable rate tbh.

Reply:
Right, I already changed it to 100 hours once but somehow it was probably reverted accidentally. Actually I am not totally sure about this, my other thought was that maybe these stats should expire even when the client is not running (expired with `time.UnixNano`). The 1000 hours value was chosen for this version originally. The alternative is the current version which only expires when the client is active. If we choose this version then we should indeed lower the constant a lot. Do you have an opinion on which version is better? Now that I think it through again, maybe I would prefer the "expire when offline" version because the relevance of the stats depends on the server churn which is independent of the client being active.

Reply:
I agree with the "expire when offline" version. Not sure from the implementation wise, how complicated it is.

Reply:
Not at all, I'll do it tomorrow.

Reply:
done

Reply:
Can we change the `expirationRate` to `72 hour?` The statistics for last 3 days should be enough.

A very high expiration rate can lead to the `uint64 overflow` since we always accumulate the metrics data.

Reply:
Each request added to the stats is scaled by `basketFactor` (currently 1000000) while `uint64` can handle more than 10^19 so we can still handle 10^13 requests. I think that is a lot but if you are worried we can reduce `basketFactor` to 1000, it does not have to be super precise.

Reply:
About the expiration: now the stats are expired also when the client is offline and I would not want to expire so fast when offline. Remembering servers from weeks ago is still useful, I think they don't come and go that quickly. On the other hand if we are online we can expect to see active servers most of the time so we could expire faster when online. It would be a trivial change to use a different constant for online and offline expiration. Like 100 vs. 1000 hours. It also helps with the accumulated amounts because they are only accumulated when online (though I don't think that is really an issue). Do you like this idea?

Reply:
Yes, I like this idea. The data is only accumulated online, a faster expiration is helpful to avoid overflow. Offline mode can have a slower rate. Let's do it.

Reply:
done